config.py

```diff
@@ -410,6 +410,72 @@
             '.*[0-9]+.*',
         ],
     ),
+
+    nb = dict(
+        lang_name = 'Norsk (bokmål)',
+        lang_dir = 'ltr',
+        database = 'nowiki_p',
+        wikipedia_domain = 'no.wikipedia.org',
+
+        beginners_link = 'https://no.wikipedia.org/wiki/Wikipedia:Bruk_av_kilder',
+        beginners_link_title = 'Wikipedia:Bruk av kilder',
+        lead_section_policy_link = 'https://no.wikipedia.org/wiki/Wikipedia:Bruk_av_kilder#Hvorfor_siterer_vi_kilder',
+        lead_section_policy_link_title = 'Wikipedia:Bruk_av_kilder',
+
+        citation_needed_category = 'Artikler som trenger referanser',
+        citation_needed_templates = EN_CITATION_NEEDED_TEMPLATES + [
+            'Trenger referanse',
+            'Tr',
+            'Referanse',
+        ],
+        citation_needed_template_name = 'trenger referanse',
+        wikilink_prefix_blacklist = EN_WIKILINK_PREFIX_BLACKLIST,
+        tags_blacklist = EN_TAGS_BLACKLIST,
+        templates_blacklist = EN_TEMPLATES_BLACKLIST,
+        hidden_category = 'Skjulte kategorier',
+        category_name_regexps_blacklist = [
+            '.*[0-9]+.*',
+        ],
+    ),
+
+    nn = dict(
```

Comment:
It looks like we [don't have a translation](https://translatewiki.net/w/i.php?title=Special:Translate&filter=%21translated&group=citationhunt&task=custom&language=nn&action=translate) for this one yet, right? Can you take care of that please? I did manage to try out the config locally, it it looks mostly fine.

However, I ended up with only 579 snippets. This is not a problem in itself, but maybe there are other templates, or alternate spellings of the existing templates, that can be added to `citation_needed_templates`? Sorry these have to be specified explicitly for now, I haven't got around to working on #51 yet :(